# gazebo robot model 
string robot_model
---
bool success
string msg
